homeassistant/components/poolsense/config_flow.py

```diff
@@ -0,0 +1,71 @@
+"""Config flow for PoolSense integration."""
+import logging
+
+from poolsense import PoolSense
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class PoolSenseConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for PoolSense."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    _options = None
+
+    def __init__(self):
+        """Initialize PoolSense config flow."""
+        self._email = None
+        self._password = None
+        self._errors = {}
```

Comment:
We don't need to store these variables as instance attributes. They are only used in the user step.